packages/ng/numeric-badge/numeric-badge.component.ts

```diff
@@ -16,11 +16,11 @@ import { NgClazz, Palette } from '@lucca-front/ng/core';
 export class NumericBadgeComponent implements OnChanges {
 	#ngClass = inject(NgClazz);
 
-	@Input({ required: true, transform: numberAttribute })
+	@Input({ required: true })
```

Comment:
I don't understand this change, this is a component made to display numbers, the `numberAttribute` seems like a perfect fit. https://angular.io/api/core/numberAttribute

Reply:
Alright ! Maybe these examples could be integrated in the story to give a better understanding of the favored use cases ?

Reply:
Ping @jeremie-lucca

Reply:
Missed that one, sorry! 
Indeed, we have multiple feedback that requires to add extra characters which lead UI devs to switch to HTML component.
And agree, I'll add extra documentation on Prisme.